### PR TITLE
feat(web): Askur chat bot on frontpage

### PR DESCRIPTION
**Pull request by RunarVestmann**
# Askur chat bot on frontpage

## What

* Added a chat bot to the frontpage of island.is
* Added specific config for a couple of articles (removing the chat bot homescreen and so forth)
* Refactored the config by adding enums and a reusable function

## Why

* This was a requested feature

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/screens/Article/components/ArticleChatPanel/config.ts [hunk 1]
```diff
@@ -12,262 +12,306 @@ export const liveChatIncConfig: Record<string, LiveChatIncChatPanelProps> = {
...
+}
+
+enum WatsonIntegration {
+  ASKUR = 'b1a80e76-da12-4333-8872-936b08246eaa',
+  ASKUR_SYSLUMENN = '0c96e8fb-d4dc-420e-97db-18b0f8bb4e3f',
+}
+
+enum WatsonServiceInstance {
+  ASKUR = 'bc3d8312-d862-4750-b8bf-529db282050a',
+}
```
Comment (kristofer-lp8f) [inline apps/web/screens/Article/components/ArticleChatPanel/config.ts:47]: Hard-coded IDs 😦 

Do these belong in CMS or separate constants file?

File: apps/web/screens/Article/components/ArticleChatPanel/config.ts [hunk 2]
```diff
@@ -12,262 +12,306 @@ export const liveChatIncConfig: Record<string, LiveChatIncChatPanelProps> = {
...
+
+enum Font {
+  IBM_PLEX_SANS = '"IBM Plex Sans", "Open Sans", Arial, sans-serif',
+}
```
Comment (kristofer-lp8f) [inline apps/web/screens/Article/components/ArticleChatPanel/config.ts:51]: Should this belong in a central fonts file?

File: apps/web/screens/Article/components/ArticleChatPanel/config.ts [hunk 3]
```diff
@@ -12,262 +12,306 @@ export const liveChatIncConfig: Record<string, LiveChatIncChatPanelProps> = {
   },
 }
 
+const setupOneScreenWatsonChatBot = (
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  instance: any,
```
Comment (kristofer-lp8f) [inline apps/web/screens/Article/components/ArticleChatPanel/config.ts:17]: `any` type 😦 

Is it possible to set the correct type?

File: apps/web/screens/Article/components/ArticleChatPanel/config.ts [hunk 4]
```diff
@@ -12,262 +12,306 @@ export const liveChatIncConfig: Record<string, LiveChatIncChatPanelProps> = {
...
+  categoryTitle: string,
+  categoryGroup: string,
+) => {
+  if (sessionStorage.getItem(categoryGroup) !== categoryTitle) {
+    sessionStorage.clear()
+  }
+  sessionStorage.setItem(categoryGroup, categoryTitle)
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const preSendhandler = (event: any) => {
```
Comment (kristofer-lp8f) [inline apps/web/screens/Article/components/ArticleChatPanel/config.ts:27]: `any` type 😦 

Is it possible to set the correct type?

File: apps/web/screens/Home/config.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+export const watsonConfig: WatsonChatPanelProps = {
+  integrationID: 'b1a80e76-da12-4333-8872-936b08246eaa',
+  region: 'eu-gb',
+  serviceInstanceID: 'bc3d8312-d862-4750-b8bf-529db282050a',
+  showLauncher: false,
+  carbonTheme: 'g10',
+  cssVariables: {
+    'BASE-font-family': '"IBM Plex Sans", "Open Sans", Arial, sans-serif',
+  },
```
Comment (kristofer-lp8f) [inline apps/web/screens/Home/config.ts:11]: Can this be shared with the `Font` above?

File: apps/web/screens/Home/config.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import { WatsonChatPanelProps } from '@island.is/web/components'
+
+export const watsonConfig: WatsonChatPanelProps = {
+  integrationID: 'b1a80e76-da12-4333-8872-936b08246eaa',
```
Comment (kristofer-lp8f) [inline apps/web/screens/Home/config.ts:4]: Can this be shared with the config above?

**Review by kristofer-lp8f — CHANGES_REQUESTED** [review]
Just some questions for improvement.

File: apps/web/screens/Article/components/ArticleChatPanel/config.ts (hunk 2 above)
Comment (RunarVestmann) [reply apps/web/screens/Article/components/ArticleChatPanel/config.ts]: I moved this to the same file as the watson integration id's live

File: apps/web/screens/Article/components/ArticleChatPanel/config.ts (hunk 4 above)
Comment (RunarVestmann) [reply apps/web/screens/Article/components/ArticleChatPanel/config.ts]: I created an interface above to at least help with the typing a bit

**Review by kristofer-lp8f — APPROVED** [review]

**Pull request merged**